juniper_official/oam/ethernet-oam-lfm.rule

```diff
@@ -27,6 +27,26 @@ healthbot {
                 }
                 type string;
                 description "Discovery state for a given interface";
+                enumerate Passive Wait as 0;
+                enumerate Send Any as 1;
+                enumerate Send Local Remote as 1;
+                enumerate Send Local Remote Ok as 3;
+            }
+            field lfmd-status {
```

Comment:
New field not used in triggers are not stored in TSDB. Why are these being added?